lib/omniauth/strategies/bitbucket.rb

```diff
@@ -4,23 +4,11 @@
 module OmniAuth
   module Strategies
     class Bitbucket < OmniAuth::Strategies::OAuth2
-      EMAIL = "email".freeze
-      ACCOUNT = "account".freeze
-
       option :client_options,
              site: "https://bitbucket.org",
              authorize_url: "/site/oauth2/authorize",
              token_url: "/site/oauth2/access_token"
 
-      def authorize_params
```

Comment:
Can you also update the README to reflect these changes?

> Notice that we'll always inject account and emails scopes, so we can retrieve the required information.

We need to clarify what's the default behavior now (it fails, it assumes a certain scope, etc).